ETL/dynamodb-etl.sh

```diff
@@ -179,6 +188,7 @@ QUERY
 
 : "${ALL:=}"
 : "${MAX_ITEMS:=25}"
+: "${MULTIPLE:=100}"
```

Comment:
This should probably default to 1 if `ALL` is not defined, otherwise `--total` will not do the right thing.